synthesizer/sed.py

```diff
@@ -64,28 +71,39 @@ def __init__(self, lam, lnu=None, description=False):
         self.broadband_fluxes = None
 
     def __add__(self, second_sed):
+
+        """
+        Overide addition operator to allow two Sed objects to be added
+        together.
+        """
+
         if not np.array_equal(self._lam, second_sed._lam):
-            exceptions.InconsistentAddition("Wavelength grids must be identical")
+            exceptions.InconsistentAddition("Wavelength grids \
+                                            must be identical")
 
         else:
             if self._lnu.ndim != second_sed._lnu.ndim:
-                exceptions.InconsistentAddition("SEDs must have same dimensions")
+                exceptions.InconsistentAddition("SEDs must have \
+                                                same dimensions")
 
             elif self._lnu.ndim == 1:
                 # if single Seds simply add together and return.
                 return Sed(self._lam, lnu=self._lnu + second_sed._lnu)
 
             elif self._lnu.ndim == 2:
-                # if array of Seds concatenate them. This is only relevant for particles.
-                return Sed(self._lam, np.concatenate((self._lnu, second_sed._lnu)))
+                # if array of Seds concatenate them.
+                # This is only relevant for particles.
+                return Sed(self._lam, np.concatenate((self._lnu,
```

Comment:
See issue #317

Reply:
Is this fixed now?

Reply:
Indeed but probably causing issues down stream. I'm in clean up mode from this change.

Reply:
OK, probably needs a more thorough test suite.